pkg/api/kubegraph/edges.go

```diff
@@ -237,7 +237,7 @@ func AddHPAScaleRefEdges(g osgraph.Graph) {
 		switch groupVersionResource.GroupResource() {
 		case kapi.Resource("replicationcontrollers"):
 			syntheticNode = kubegraph.FindOrCreateSyntheticReplicationControllerNode(g, &kapi.ReplicationController{ObjectMeta: syntheticMeta})
-		case deployapi.Resource("deploymentconfigs"):
+		case deployapi.Resource("deploymentconfigs"), deployapi.LegacyResource("deploymentconfigs"):
```

Comment:
Was this added late to the branch? Cannot find it yet in the version in the rebase.